test/parallel/test-abort-uncaught-exception.js

```diff
@@ -0,0 +1,30 @@
+'use strict';
+
+const assert = require('assert');
+const exec = require('child_process').exec;
+const node = process.execPath;
+
+if (process.argv[2] === 'child') {
+  setImmediate(function() {
+    throw new Error('child error');
+  });
+} else {
+  run('', null, 1);
+  run('--abort-on-uncaught-exception', 'SIGABRT', 134);
+}
+
+function run(flags, signal, code) {
+  let cmd = [node, flags, __filename, 'child'].join(' ');
+  exec(cmd, function(err, stdout, stderr) {
+    assert(err);
+    // darwin only provides the signal on an abort
+    // linux only provides the exit code (134) on an abort
```

Comment:
This isn't right.  Both platforms raise SIGABRT on `abort()` but SIGABRT has a different signal number on Linux than on OS X, 6 vs. 22.  When a process terminates after a signal, the exit code is `128 + signo`, so 134 on Linux and 150 on OS X.